packages/core/saved-objects/core-saved-objects-api-browser/src/apis/bulk_create.ts

```diff
@@ -12,6 +12,7 @@ import type { SavedObjectsCreateOptions } from './create';
  * Per-object parameters for bulk create operation
  *
  * @public
+ * @deprecated See https://github.com/elastic/dev/issues/2194
```

Comment:
I opted for an internal issue because I wanted a place to share internal resources. I've opened https://github.com/elastic/kibana/issues/149098 with public-facing info and then we can keep https://github.com/elastic/dev/issues/2194 for internal resources.